bin/_config.sh

```diff
@@ -42,10 +42,13 @@ function _set_openshift_env() {
 #
 # usage: _docker_run [OPTIONS] [COMMAND]
 #
-#   OPTIONS: docker run options
-#    --env-file: file to set environment variables (_e.g._ env.d/production)
-#
-#   COMMAND: command to be run in Arnold container (_e.g._ ansible-playbook deploy.yml)
+#   OPTIONS:
+#     -e    file to set environment variables (_e.g._ env.d/production)
+#     -m    mount sources as a volume
```

Comment:
You should provide an example. I have no idea how to use this option without viewving the code